www_src/pages/project/touchhandler.js

```diff
@@ -1,69 +1,126 @@
 module.exports = function(positionable) {
-  return {
+
+  var resetTransform = function() {
+    return {
+      x1: false,
+      y1: false,
+      x2: false,
+      y2: false,
+      distance: 0,
+      angle: 0
+    };
+  };
+
+  var transform = resetTransform();
+  var mark = JSON.parse(JSON.stringify(positionable.state));
```

Comment:
not in this case. Assign preserves references, and in this case I need guaranteed "these are new values, completely detached from any and all preexisting bindings".